ros2launch/ros2launch/api/api.py

```diff
@@ -159,10 +159,12 @@ def launch_a_launch_file(
     if args and args.launch_prefix_filter:
         launch_file_arguments.append(f'launch-prefix-filter:={args.launch_prefix_filter}')
 
+    log_file_name = args.log_file_name if args and args.log_file_name else 'launch'
```

Comment:
```suggestion
    log_file_name = 'launch'
    if args and args.log_file_name:
      log_file_name = args.log_file_name  
```
This is a nitpick but I find inline ifs hard to read.